stub-runner/stub-runner/build.gradle

```diff
@@ -9,6 +9,8 @@ dependencies {
     compile 'org.apache.ivy:ivy:2.4.0'
     compile 'org.codehaus.groovy:groovy-all'
     compile 'com.github.tomakehurst:wiremock'
+    compile 'org.eclipse.jetty:jetty-servlet'
+    compile 'org.eclipse.jetty:jetty-servlets'
```

Comment:
add compile 'log4j:log4j:1.2.17' to remove processing failed message.. also in micro-deps/gragle I guess